src/TemplateNPC.cpp

```diff
@@ -534,698 +602,665 @@ bool sTemplateNPC::CanEquipTemplate(Player* player, std::string& playerSpecStr)
+        static std::vector<ChatCommand> TemplateNPCTable = {
+            {"copy", SEC_ADMINISTRATOR, false, &HandleCopyCommand, "Copies yours target gear onto your character. example: `.template copy`"},
+            {"save", SEC_ADMINISTRATOR, false, nullptr, "", saveTable},
+            {"reload", SEC_ADMINISTRATOR, true, &HandleReloadTemplateNPCCommand, ""},
+        };
+
+        static std::vector<ChatCommand> commandTable = {
+            {"template", SEC_ADMINISTRATOR, true, nullptr, "", TemplateNPCTable},
         };
+
         return commandTable;
     }
 
+    static bool HandleCopyCommand(ChatHandler* handler, const char* _args)
+    {
+        Player* caller = handler->GetSession()->GetPlayer();
+        if (!caller)
+        {
+            assert(false);
+        }
+        uint64 selected = handler->GetSession()->GetPlayer()->GetTarget();
+        if (!selected)
+        {
+            handler->SendGlobalGMSysMessage("Need to select a player");
+            return false;
+        }
+        Player* target = ObjectAccessor::FindPlayerInOrOutOfWorld(selected);
+
+        if (!target)
+        {
+            handler->SendGlobalGMSysMessage("Need to select a player");
+            return false;
+        }
+
+        if (caller == target)
+        {
+            handler->SendGlobalGMSysMessage("You are trying to copy your own gear, which is pointless.");
+            return false;
+        }
+
+        if (caller->getClass() != target->getClass())
+        {
+            handler->SendGlobalGMSysMessage("In order to copy someone you need to be of the same class.");
+            return false;
+        }
+
+        sTemplateNpcMgr->Copy(caller, target);
+
+        handler->SendGlobalGMSysMessage("Character coppied.");
+
+        return true;
+    }
+
+    static bool HandleSaveGearHorde(ChatHandler* handler, const char* _args)
+    {
+        return HandleSaveCommon(handler, _args, TEMPLATE_HORDE);
+    }
+
+    static bool HandleSaveGearAlly(ChatHandler* handler, const char* _args)
+    {
+        return HandleSaveCommon(handler, _args, TEMPLATE_ALLIANCE);
+    }
+
+    static bool HandleSaveGearHuman(ChatHandler* handler, const char* _args)
+    {
+        return HandleSaveCommon(handler, _args, TEMPLATE_HUMAN);
+    }
+
+    static bool HandleSaveCommon(ChatHandler* handler, const char* _args, TemplateType type)
+    {
+        if (!*_args)
+            return false;
+
+        auto spec = std::string(_args);
+
+        auto selectedSpec = knownSpecs.find(spec);
+        if (selectedSpec == knownSpecs.end())
+        {
+            handler->SendGlobalGMSysMessage("Unknown spec passed");
+            return false;
+        }
+
+        Player* p = handler->getSelectedPlayerOrSelf();
+        if (p == nullptr)
+            return false;
+
+        sTemplateNpcMgr->ExtractGearTemplateToDB(p, spec, type);
+
+        handler->SendGlobalGMSysMessage("Template extracted to DB. You might want to `.template reload`.");
```

Comment:
```suggestion
        handler->SendGlobalGMSysMessage("Template extracted to DB. You might want to type \".template reload\".");
```

I don't think the backticks will show differently ingame, so i changed slightly ^^